src/category_theory/monad/adjunction.lean

```diff
@@ -63,18 +77,36 @@ def monad.comparison {L : C ⥤ D} {R : D ⥤ C} (h : L ⊣ R) : D ⥤ h.to_mona
 The underlying object of `(monad.comparison R).obj X` is just `R.obj X`.
 -/
 @[simps]
-def monad.comparison_forget {L : C ⥤ D} {R : D ⥤ C} (h : L ⊣ R) :
+def monad.comparison_forget (h : L ⊣ R) :
   monad.comparison h ⋙ h.to_monad.forget ≅ R :=
 { hom := { app := λ X, 𝟙 _, },
   inv := { app := λ X, 𝟙 _, } }
 
+lemma monad.left_comparison (h : L ⊣ R) : L ⋙ monad.comparison h = h.to_monad.free := rfl
```

Comment:
Equality of functors that's `rfl` is typically okay, but I don't mind changing it

Reply:
More specifically, when taking components of either side of this, `dsimp` will say they're the same; I wanted to add this lemma more so that it confirms the properties of `monad.comparison`